src/extensions/default/HealthData/HealthDataManager.js

```diff
@@ -110,8 +110,8 @@ define(function (require, exports, module) {
     }
 
     /*
-     * Check if the health data is to be sent to the server. If the user has enabled tracking, health data will be sent once every 24 hours.
-     * Send health data to the server if the period is more than 24 hours.
+     * Check if the Health Data is to be sent to the server. If the user has enabled tracking, Health Data will be sent once every 24 hours.
+     * Send Health Data to the server if the period is more than 24 hours.
```

Comment:
I don't know if we need to capitalize every place we use the phrase "health data," since it's not the official name of the feature or anything. I could see capitalizing (part of) the phrase "Health Report data" since that matches the user-facing feature name.  Not a big deal either way though.